save-backend/src/test/kotlin/org/cqfn/save/backend/controller/TestInitializeControllerTest.kt

```diff
@@ -104,10 +109,10 @@ class TestInitializeControllerTest {
             .exchange()
             .expectStatus()
             .isOk
-            .expectBody<List<TestDto>>()
+            .expectBody<List<TestDtoForBatch>>()
             .consumeWith {
                 println(it.responseBody)
-                assertTrue(it.responseBody!!.isNotEmpty() && it.responseBody!!.size == 1)
+                assertTrue(it.responseBody!!.isNotEmpty())
```

Comment:
Why did you remove an assertion on size?

Reply:
I retured it back after @aktsay6 fixed it